apps/sms/test/marionette/lib/messages.js

```diff
@@ -9,6 +9,26 @@
     BACKSPACE: '\ue003'
   };
 
+  function observeElementStability(el) {
+    delete el.dataset.__stable;
+
+    function markElementAsStable() {
+      return setTimeout(function() {
+        el.dataset.__stable = 'true';
+        observer.disconnect();
+      }, 1000);
```

Comment:
ya I know there might be several changes, but doesn't each change trigger the observer callback? And each callback will reset the timeout again. Or the observer only callback once even we add 4 recipients?